angular.json

```diff
@@ -96,6 +96,7 @@
   "cli": {
     "schematicCollections": [
       "@ngrx/schematics"
-    ]
+    ],
+    "analytics": false
```

Comment:
Why do you need this?

Reply:
it prevents angular from sending analytics to google (was added automatically on ng serve)